docs/how-to-guides/rook-ceph-storage.md

```diff
@@ -0,0 +1,291 @@
+# Kubernetes storage with Rook Ceph on Packet cloud
+
+## Contents
+
+- [Introduction](#introduction)
```

Comment:
This diverges from the format that we have for the how-to-guides:

https://github.com/kinvolk/lokomotive/blob/master/docs/how-to-guides/authentication-with-dex-gangway.md#contents
https://github.com/kinvolk/lokomotive/blob/master/docs/how-to-guides/cloudflare-dns.md#contents

I'd stay consistent with the content structure.

Reply:
I have made changes to the structure, PTAL :-)